fixes.inc

```diff
@@ -14281,6 +14354,133 @@ native BAD_DestroyActor(actorid) = DestroyActor;
 	#define DestroyActor__( FIXES_DestroyActor(
 #endif
 
+/**
+ * <fixes> Various bugs in GetVehicleComponentInSlot function.</fixes>
+ */
+
+#if _FIXES_SAMP && defined _ALS_GetVehicleComponentInSlot
+	#error _ALS_GetVehicleComponentInSlot defined
+#endif
+
+native BAD_GetVehicleComponentInSlot(vehicleid, slot) = GetVehicleComponentInSlot;
+
+#if FIX_GetVehicleComponentInSlot
+
+	stock FIXES_GetVehicleComponentType(componentid)
+	{
+		if(IsComponentFrontBullbar(componentid))
```

Comment:
This won't work, you need to reference `FIXES_IsComponentFrontBullbar`.

Reply:
Fixed.